code/modules/fabrication/designs/robotics/designs_prosthetics.dm

```diff
@@ -1,6 +1,45 @@
+#define DEFINE_ROBOLIMB_DESIGNS(MODEL_PATH, MODEL_ID, MODEL_NAME)          \
+/datum/fabricator_recipe/robotics/prosthetic/model_##MODEL_ID {            \
+	category = MODEL_NAME + " Prosthetics";                                \
```

Comment:
Sadly if I make these indented with spaces so the `\\` lines up, it trips the incorrect indentation detection. :(